ChangeLog.md

```diff
@@ -20,6 +20,9 @@ See docs/process.md for more on how version tagging works.
 
 2.0.23
 ------
+- emcmake (or projects otherwise using Emscripten.cmake) now install directly
+  into emscripten's sysroot by default.  This means that for many use cases
+  CMAKE_INSTALL_PREFIX is no longer needed.
```

Comment:
I'm not sure what this means. That `make install` will copy to the sysroot? Or something else?

Reply:
Yes.

Previously the default was to try to install to `/usr/local` on the host system.. which would either give you "Permission Denied" or install into a place you probably never want to store emscripten libraries.

Reply:
I will try to elaborate.

Reply:
I wrote a much longer ChangeLog.md entry.

I think I will wait until we release 2.0.23 before landing since I'd like to get the std libs updates into their own separate release.